### DISCUSSION TITLE
Openvpn - i have a lot of servers - it would be nice to see ping rate listed next to each server, updated every hour or so

### DISCUSSION THREAD

**Discussion by frankozland** [RaspAP/raspap-webgui#1642] [Ideas]
I have a lot of servers configured for openvpn.  Its difficult to figure out which one has the best ping rate (and hopefully bandwidth).  
![image](https://github.com/user-attachments/assets/5b1b417d-0681-4ada-9c00-4184a6770ad0)

I put together a quick and dirty perl script that can get the remote host ip address and ping it, logging results to /tmp.

[ping_all_openvpn_servers.pl.txt](https://github.com/user-attachments/files/16655726/ping_all_openvpn_servers.pl.txt)

**Comment by frankozland** [comment]
Heres what the output looks of the script - its just a csv of client config files, the ipaddress found inside, and the ping response rate.
![image](https://github.com/user-attachments/assets/d7dfd47b-c08c-41e6-814a-2b407c657735)

**Comment by feh123** [comment]
Hi, this is a very nice idea. Thanks for publishing. I too have many configurations and currently I pick the best by seeing what my iphone choses in the relevant VPN provider app. It works but is rather slow. To be able to see straight away in the UI would be great. I use wireguard and nordvpn - would the script see both? I just wanted to check how you installed the script. You created in /etc/hostapd a file hostapd.conf.users and added the script. Then on the RaspAP UI you save settings for the hotspot and then restart it? Thanks.
